scripts/migrate_master_azs.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+set -e
+
+installation=$1
+cluster=$2
+
+read -p "Installation: $installation, Cluster: $cluster.
+Press enter to continue, ctrl+c to abort"
+
+opsctl create kubeconfig -i $installation
+gsctl create kubeconfig --cluster=$cluster --certificate-organizations system:masters -e $installation
+
+# get org name
+org_namespace="$(kubectl --context=giantswarm-$installation -n org-giantswarm get cluster -l giantswarm.io/cluster=${cluster} -o yaml -A|yq -r '.items[0].metadata.namespace')"
+org="$(kubectl --context=giantswarm-$installation -n org-giantswarm get cluster -l giantswarm.io/cluster=${cluster} -o yaml -A|yq -r '.items[0].metadata.labels["giantswarm.io/organization"]')"
+
+# get azure operator version
+azure_operator_version="$(kubectl --context=giantswarm-$installation -n ${org_namespace} get cluster ${cluster} -o yaml| yq -r '.metadata.labels["azure-operator.giantswarm.io/version"]')"
+
+# find subscription ID
+subscription="$(kubectl --context=giantswarm-$installation -n giantswarm get secret -l giantswarm.io/organization="${org}" -o yaml | yq -r '.items[0].data["azure.azureoperator.subscriptionid"]'| base64 -d)"
+
+master_ip="$(kubectl --context=giantswarm-$cluster get no $cluster-master-$cluster-000000 -o yaml| yq -r '.status.addresses[] | select(.type == "InternalIP").address')"
```

Comment:
I'm not sure if it's always guaranteed that master node has `-000000` suffix. Could be better to rely on `role=master` label?

Reply:
thanks, done